packages/anoncreds/src/AnonCredsApi.ts

```diff
@@ -179,6 +181,10 @@ export class AnonCredsApi {
     }
   }
 
+  public async getCreatedSchemas(query: SimpleQuery<AnonCredsSchemaRecord>) {
```

Comment:
I think this (and the credential definition counterpart) are some nice hidden gems of this PR :laughing: